omero/sysadmins/version-requirements.txt

```diff
@@ -1425,12 +1456,12 @@ Ice
       - `Ice 3.5 <http://www.zeroc.com/forums/announcements/5943-ice-3-5-0-released.html>`__
         `Ice 3.5.1 <http://www.zeroc.com/forums/announcements/6118-ice-3-5-1-released.html>`__
     * - 3.6
-      - TBA (in beta)
-      - TBA
-      - Unsupported
+      - from June 2015
+      - to TBA
       - Unsupported
+      - Upcoming
       - Recommended
-      -
+      - `Ice 3.6.0 <http://zeroc.com/forums/announcements/6468-ice-3-6-0-ice-touch-3-6-0-released.html>`__
```

Comment:
https://forums.zeroc.com/forum/announcements/6631-ice-3-6-0-and-ice-touch-3-6-0-released
Your link is just redirecting to the top level of the forum